pkg/syncmap/syncmap_test.go

```diff
@@ -131,3 +133,12 @@ func (suite *SyncMapUtilSuite) TestGetAsTime(c *check.C) {
 	result, _ := mmap.GetAsTime("aaa")
 	c.Check(result, check.DeepEquals, expected)
 }
+
+func (suite *SyncMapUtilSuite) TestGetAsAtomicInt(c *check.C) {
+	expected := atomiccount.NewAtomicInt(10)
+	mmap := NewSyncMap()
+	mmap.Add("aaa", expected)
+
+	result, _ := mmap.GetAsAtomicInt("aaa")
+	c.Check(result, check.DeepEquals, expected)
```

Comment:
You could add code here to check the non-existent one, then check the non-nil error:
```
result, err := mmap.GetAsAtomicInt("nonexist")
```

Reply:
updated, PTAL